packages/node/src/models/proposed-app-instance-info.ts

```diff
@@ -60,10 +67,10 @@ export class ProposedAppInstanceInfo implements AppInstanceInfo {
   intermediaries?: Address[];
 
   constructor(
-    appInstanceId: AppInstanceID,
-    proposeParams: IProposedAppInstanceInfo
+    proposeParams: IProposedAppInstanceInfo,
+    channel?: StateChannel,
+    overrideId?: Bytes32
```

Comment:
What is an override ID?

Reply:
Used in `fromJson` when you don't have the channel available and need to set the id field